frontend/playground-frontend/src/components/LearnMod/LearnMod.tsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import ClassCard from "./ClassCard";
+import getContent from "./LearningModulesContent";
+import { useAppSelector } from "../../redux/hooks";
+//import { fetchUserProgressData } from "../../redux/userLogin";
```

Comment:
commented out code?

Reply:
I disabled fetching user progress data until Sebastian fixes the structure of the json for user progress data